resources/sdk/webmessagingjava/config.json

```diff
@@ -14,7 +14,7 @@
 		"swagger": {
 			"internalSwaggerPath": "https://api.inindca.com/api/v2/docs/swaggerall",
 			"oldSwaggerPath": "${SDK_REPO}/swagger.json",
-			"newSwaggerPath": "https://s3.amazonaws.com/inin-prod-use1-purecloud-shared/webmessaging/swagger/swagger-latest.json",
+			"newSwaggerPath": "https://s3.amazonaws.com/inin-dev-use1-purecloud-shared/webmessaging/swagger/swagger-latest.json",
```

Comment:
Public SDKs must be built against production. You can use an alternate swagger definition for the internal SDK in the SDK config repo in bitbucket.

Reply:
on which repo I should include DCA ? there are some definition not yet in production